packages/frontend/src/features/comments/components/DashboardTileComments.tsx

```diff
@@ -31,6 +41,60 @@ export const DashboardTileComments: FC<
 
     const { mutateAsync, isLoading } = useCreateComment();
 
+    const { data: notifications } = useGetNotifications(
+        NotificationResourceType.DashboardComments,
+        true,
+    );
+    const { mutate: updateNotification } = useUpdateNotification();
+
+    // TODO: This is a temporary solution. We should probably have a separate endpoint for unread comments
+    const unreadCommentsForThisTile = useMemo(
+        () =>
+            notifications?.filter(
+                (n) =>
+                    !n.viewed &&
+                    n.metadata?.dashboardTileUuid === dashboardTileUuid &&
+                    n.metadata?.dashboardUuid === dashboardUuid,
+            ),
+        [notifications, dashboardTileUuid, dashboardUuid],
+    );
+
+    const showIndicator = comments && comments.length > 0;
+    const indicatorColor = useMemo(() => {
+        if (unreadCommentsForThisTile && unreadCommentsForThisTile.length > 0)
```

Comment:
naming `unreadCommentsForThisTile` is confusing, because it relates to `mentions` , not just comments, right ?

Reply:
oops, yes 😆  will rename

Reply:
I renamed to `unreadNotificationsForTile` because `unreadMentionNotificationsForTile` was too long IMO, lmk what you think